js_13/js/script.js

```diff
@@ -2,76 +2,71 @@
 
 $(function(){
 
-  if (localStorage.getItem('sectionQA')) {
-
-    var sectionQA = localStorage.getItem('sectionQA');
+  if (!localStorage.getItem('questions')) { return; }
 
+    var questions = JSON.parse(localStorage.getItem('questions'));
     var html = $('#content-template').html();
-
     var tmpl = _.template(html);
-    var content = tmpl(sectionQA);
+    var content = tmpl(questions);
     
     $('.content').append(content);
     
-    $('.btn').on('click', function(){
-      var correctAnswers = JSON.parse(localStorage.getItem('correctAnswers'));
-      var n = 3;
+    $('.btn').on('click', function(){      
+      showModal(defineResults('.answers li input'));
+
+      $('.result-modal .btn-close').on('click', function () {
+        hideModal();
+      });
+      
+    });
+    
+    function defineResults(listElement) {
```

Comment:
if you passing the string, it's better to name parameter like `listElementSelector` to make if more obvious